### PR TITLE
Port #2610 and tagging to master

### PR DESCRIPTION
**Pull request by juliogonzalez**
## What does this PR change?

Port #2610 and tagging to master

### CODE REVIEW

File: java/spacewalk-java.spec [hunk 1]
```diff
@@ -64,7 +64,7 @@ Name:           spacewalk-java
 Summary:        Java web application files for Spacewalk
 License:        GPL-2.0-only
 Group:          Applications/Internet
-Version:        4.1.16
```
Comment (juliogonzalez) [inline java/spacewalk-java.spec:67]: Not sure why this is happening. On master the version is *already 4.2.0*

File: spacewalk/admin/spacewalk-admin.spec [hunk 2]
```diff
@@ -28,7 +28,7 @@ License:        GPL-2.0-only
 Group:          Applications/Internet
 Name:           spacewalk-admin
 Url:            https://github.com/uyuni-project/uyuni
-Version:        4.1.5
```
Comment (juliogonzalez) [inline spacewalk/admin/spacewalk-admin.spec:31]: Same

File: spacewalk/setup/spacewalk-setup.spec [hunk 3]
```diff
@@ -36,7 +36,7 @@
 %{!?fedora: %global sbinpath /sbin}%{?fedora: %global sbinpath %{_sbindir}}
 
 Name:           spacewalk-setup
-Version:        4.1.5
```
Comment (juliogonzalez) [inline spacewalk/setup/spacewalk-setup.spec:39]: And same

**Review by moio — COMMENTED** [review]
I suggest rebasing this branch on top of master to get a clean set of commits

**Comment by juliogonzalez** [comment]
> I suggest rebasing this branch on top of master to get a clean set of commits

If I am not wrong, doing this will require the history, meaning the tags won't work on master.

If that's the case, that's exactly why we do this, and why we merge such kind of PRs with "Create a merge commit" and not with "rebase and merge"

**Comment by moio** [comment]
> If I am not wrong, doing this will require the history, meaning the tags won't work on master.
> 
> If that's the case, that's exactly why we do this, and why we merge such kind of PRs with "Create a merge commit" and not with "rebase and merge"

You are right, sorry, I thought that comment got deleted because I had figured out the same myself. Apparently I mis-clicked and it still got posted.

A better solution is either to handcraft a commit that prevents merge conflicts (like you did), or even better just fix in the merge commit itself (but that is only possible via CLI `git`, IIUC).

Sorry for the noise

**Pull request merged**